src/main/kotlin/org/jitsi/jibri/service/impl/StreamingJibriService.kt

```diff
@@ -96,6 +97,7 @@ class StreamingJibriService(
         registerSubComponent(FfmpegCapturer.COMPONENT_ID, capturer)
     }
 
+    @ExperimentalTime
```

Comment:
Instead of putting these annotations everywhere, you can add a new line `<arg>-Xopt-in=kotlin.time.ExperimentalTime</arg>` after [this line](https://github.com/jitsi/jibri/blob/master/pom.xml#L296) in the pom.xml.

Reply:
Thank you! Done